### ISSUE TITLE
is this safe

### ISSUE THREAD

**Issue by 2115lovesuGenghis** [KanekiWeb/Nitro-Generator#26]
ive just seen a lot of nitro gens that seem sketchy so i just want to see if this one is safe before i download it

**Comment by CatelHd** [comment]
yes,but its just cap,dw

**Comment by GachaYTB** [comment]
yes, but the program also will send a message to the owner telling that a nitro is valid.

**Comment by 3pm-on-github** [comment]
yes, it is safe.
i verified it to see if its safe and it is.
it also verifies the link
